cli/azd/cmd/actions/action.go

```diff
@@ -17,8 +17,9 @@ func (a ActionFunc) Run(ctx context.Context) (*ActionResult, error) {
 
 // Define a message as the completion of an Action.
 type ResultMessage struct {
-	Header   string
-	FollowUp string
+	Header        string
+	CorrelationId string
```

Comment:
This seems wrong...? Should a `CorrelationId` be part of an ActionResult?

Reply:
I don't think we should be outputting a "provision" correlation ID in the "up" flow. It seems hard for a user to guess that the correlation ID belonged to the infrastructure provisioning operation out of all the things that happened. 

I'd encourage for a design in `provision` that would support logging a message that easily understood to be part of the operation.

Reply:
I will remove the correlationId from the changes.
It was requested by Rajesh!!! hahah,, and he did not hear my reasons about why it was not good to add it until you and Matt say it as well =D